src/dfx/Cargo.toml

```diff
@@ -88,6 +89,11 @@ url = "2.1.0"
 walkdir = "2.2.9"
 wasmparser = "0.87.0"
 which = "4.2.5"
+dirs-next = "2.0.0"
```

Comment:
We already have `directories-next` as a dependency.  Can we get what we need from there rather than also adding `dirs-next` ?

Reply:
The rest (or most of them) of the dependencies are in alphabetical order.  Adding this in alphabetical order might have helped.